charts/iter8lib/templates/_metrics-ce.tpl

```diff
@@ -0,0 +1,83 @@
+{{- define "metrics.ce" -}}
+# endpoint where the monitoring instance is available
+# https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints#endpoints_sysdig
+url: {{ .Values.Endpoint }}/prometheus/api/v1/query # e.g. https://ca-tor.monitoring.cloud.ibm.com
```

Comment:
Does CE use Prometheus or sysdig? It says prometheus here but `provider` below refers to sysdig

Reply:
CE uses both. I can change this. I think this is just a remnant of my early explorations.